src/idx.ts

```diff
@@ -1,5 +1,6 @@
 import type { CeramicApi } from '@ceramicnetwork/common'
 import { IDX } from '@ceramicstudio/idx'
+import { createTile, createDefinition } from '@ceramicstudio/idx-tools'
```

Comment:
Please don't use these APIs, they are internal and not documented so I'd rather avoid using them in this kind of quick start.

Reply:
Understood, I thought that the idx-tools was the most "official" way to do it. In that case is it better to include the Definition that we export in idx-constants or is there a way that conforms to our best practices?

Reply:
Yes getting the schema for idx-constants should be a good option 👍